app/javascript/tests/patient/history/HistoryList.test.js

```diff
@@ -24,7 +24,7 @@ describe('HistoryList', () => {
   it('Properly renders all main components', () => {
     const wrapper = getWrapper();
     expect(wrapper.find('#histories').exists()).toBeTruthy();
-    expect(wrapper.find(Card.Header).find('span').text()).toEqual('History');
+    expect(wrapper.find(Card.Header).at(0).text()).toContain('History');
```

Comment:
same comment about `toEqual` and `at(0)`

Reply:
```
    Expected: "History"
    Received: "History<InfoTooltip />"
```
Im leaving it as toContain

Reply:
And there's a second one at the bottom, hence the at(0)